tests/scripts/components-basic-checks.sh

```diff
@@ -18,3 +18,102 @@ components_tf_psa_crypto_check_python_files () {
     msg "Lint: Python scripts"
     $FRAMEWORK/scripts/check-python-files.sh
 }
+
+component_tf_psa_crypto_check_recursion () {
+    msg "Check: recursion.pl" # < 1s
+    $FRAMEWORK/scripts/recursion.pl ${PSA_CORE_PATH}/*.c
+    $FRAMEWORK/scripts/recursion.pl ${BUILTIN_SRC_PATH}/*.c
+}
+
+component_tf_psa_crypto_check_changelog () {
+    msg "Check: changelog entries" # < 1s
+    rm -f ChangeLog.new
+    $FRAMEWORK/scripts/assemble_changelog.py -o ChangeLog.new
+    if [ -e ChangeLog.new ]; then
+        # Show the diff for information. It isn't an error if the diff is
+        # non-empty.
+        diff -u ChangeLog ChangeLog.new || true
+        rm ChangeLog.new
+    fi
+}
+
+component_tf_psa_crypto_check_test_cases () {
+    msg "Check: test case descriptions" # < 1s
+    if [ $QUIET -eq 1 ]; then
+        opt='--quiet'
+    else
+        opt=''
+    fi
+    $FRAMEWORK/scripts/check_test_cases.py -q $opt
+    unset opt
+}
+
+component_tf_psa_crypto_check_test_dependencies () {
```

Comment:
I think we can just remove the corresponding one in mbedtls.

Reply:
Created: https://github.com/Mbed-TLS/mbedtls/pull/9914